### PR TITLE
Exposing `Curve.evaluate_hodograph()` as part of the public API.

### LINKED ISSUE #138: Expose evaluate_hodograph in Python public API
Hi!  I am trying to use `evaluate_hodograph` in Python 3.6, but have been having some substantial difficulty with importing it.  I have been through the documentation on binary extensions many times, but have been unable to figure this out.

I have tried `cimport bezier._curve` (as described here: https://bezier.readthedocs.io/en/0.9.0/python/pxd/index.html), but this also failed due to a syntax error (I presume due to the fact that I'm using Python instead of Cython).

In addition, I tried building from the source using `python3.6 setup.py build_ext`, which also did not work.  This was equivalent to installing via pip.

Do you know what the issue may be?  I am using the Ubuntu Bash shell native to Windows 10; thus far, I have been treating the system as an Ubuntu system when performing installations, which has worked well, so I don't think I will need to deal with DLL files.  If there is any other information I can provide, please let me know.  Thank you!

### PR DESCRIPTION
**Pull request by dhermes**
Fixes #138.

However, this just exposes a subroutine / procedure from the ABI. A "complete" implementation would include an equivalent `Curve.evaluate_hodograph_multi()`.

### CODE REVIEW

File: src/python/bezier/curve.py [hunk 1]
```diff
@@ -276,6 +276,38 @@ def evaluate_multi(self, s_vals):
         """
         return _curve_helpers.evaluate_multi(self._nodes, s_vals)
 
+    def evaluate_hodograph(self, s):
```
Comment (dhermes) [inline src/python/bezier/curve.py]: cc @anntzer

I am (for now) just exposing the method that already exists. There is no equivalent `evaluate_hodograph_multi()`.

Somewhat related to this is #87.

**Comment by dhermes** [comment]
@anntzer I'm not when the next release will be, but I can try to make one this week if you'd like.

**Comment by anntzer** [comment]
Thanks!  A new release would be appreciated, but it's not urgent either -- whatever works for you.

**Comment by dhermes** [comment]
FYI @anntzer the `2020.2.3` just came out. It is in many ways a **very big** release because it's the first release where the `libbezier` bits are completely foreign to the Python build system (now built via CMake). :crossed_fingers: nothing unexpected happens

**Comment by anntzer** [comment]
Thanks!

**Pull request merged**